### DISCUSSION TITLE
Heroku student account

### DISCUSSION THREAD

**Discussion by Kylefairbairn** [community/community#38308] [GitHub Education]
Anyone try to sign up for a Heroku account ?

**Comment by ossanna16** [comment]
Hi there @Kylefairbairn and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by samus-aran** [comment]
Hi @Kylefairbairn, we're happy you're here! You are more likely to get a useful response if you are explicit about what you need, giving a few more details might help someone give you a nudge in the right direction. 👍
